vb-fe/src/services/Room.ts

```diff
@@ -8,6 +8,10 @@ export const createRoom = (room: RoomForm, userId: string): Promise<any> => {
   return Http.post(`${RoomEndpoints.GENERAL}`, { room, userId });
 };
 
+export const updateRoom = (room: RoomForm, userId: string, roomId: string): Promise<any> => {
```

Comment:
no need to pass the `roomId` and `userId` separately. The roomId will be in `Room` because this is update room. The user Id will be in the header